### ISSUE TITLE
A11y > Iframe needs a title attribute

### ISSUE THREAD

**Issue by Blackbaud-SteveBrush** [blackbaud/auth-client#57]
When SPAs run `SkyA11y.run()` with the omnibar present, they generate the following error during e2e tests:
```
error: aXe - [Rule: 'frame-title'] Frames must have title attribute - WCAG: wcag241
       Get help at: https://dequeuniversity.com/rules/axe/2.6/frame-title?application=webdriverjs

       Elements:

<iframe class="sky-omnibar-iframe" src="https://host.nxt.blackbaud.com/omnibar/" cd_frame_id_="9117f7ee6c756062c36d8f45da5e9bea"></iframe>

<iframe class="sky-omnibar-iframe-session-watcher" width="0" height="0" frameborder="0" src="https://s21aidntoken00blkbapp01.nxt.blackbaud.com/SessionWatcher.html?origin=https%3A%2F%2Fhost.nxt.blackbaud.com" tabindex="-1" cd_frame_id_="6f0ec366990e7981df3177b2562af574"></iframe>
```

**We need to add a unique title attribute to the omnibar's iframe:**
https://github.com/blackbaud/auth-client/blob/master/src/omnibar/omnibar.ts#L35
